### PR TITLE
Update javadoc

### PR DESCRIPTION
**Pull request by binarynoise**
Somebody an idea for a commit message?
#790

### CODE REVIEW

File: acra-core/src/main/java/org/acra/annotation/AcraCore.java [hunk 1]
```diff
@@ -195,6 +195,12 @@
      * Provide here regex patterns to be evaluated on each {@link android.content.SharedPreferences} key to exclude KV pairs from the collected SharedPreferences.
      * This allows you to exclude sensitive user data like passwords from being collected.
      *
+     * If you only want to include some keys, use can use the following regexes:
```
Comment (mikehardy) [inline acra-core/src/main/java/org/acra/annotation/AcraCore.java:198]:
```suggestion
     * If you only want to include some keys, you may use regular expressions to do so:
```

File: acra-core/src/main/java/org/acra/annotation/AcraCore.java [hunk 2]
```diff
@@ -203,6 +209,12 @@
      * Provide here regex patterns to be evaluated on each {@link android.provider.Settings.System}, {@link android.provider.Settings.Secure} and {@link android.provider.Settings.Global} key to exclude KV pairs from being collected.
      * This allows you to exclude sensitive data from being collected.
      *
+     * If you only want to include some keys, use can use the following regexes:
```
Comment (mikehardy) [inline acra-core/src/main/java/org/acra/annotation/AcraCore.java:212]:
```suggestion
     * If you only want to include some keys, you may use regular expressions to do so:
```

**Comment by mikehardy** [comment]
Cool! Everyone has opinions on grammar, so I suggested my opinions, but I think this will make sure your question is answered for any future folks, very helpful

**Comment by binarynoise** [comment]
Thanks for fixing my grammar. I accepted the suggestions

**Pull request merged**